flexmeasures/data/models/planning/linear_optimization.py

```diff
@@ -339,10 +339,16 @@ def cost_function(m):
     model.costs = Objective(rule=cost_function, sense=minimize)
 
     # Solve
+
+    # load_solutions=False to avoid a RuntimeError exceptions in HiGHS when solving and infeasible problem.
```

Comment:
2 typos (exception singular, and -> an)

Reply:
And it's not just HiGHS. I saw a GH discussion in the Pyomo repo discussing the more general issue of (I believe appsi) solvers raising rather than silently setting the termination condition to infeasible.

So your change is probably an improvement in general, where our fallback logic would otherwise have failed when the FM host uses a different solver than CBC.

Reply:
Nice! Thanks for pointing out that this could be useful for other solvers, as well.